src/utils/types.ts

```diff
@@ -0,0 +1,14 @@
+import { METHODS } from "~src/utils/HTTPTransport";
+
+export type Nullable<T> = T | null;
+export type Keys<T extends Record<string, unknown>> = keyof T;
+export type Values<T extends Record<string, unknown>> = T[Keys<T>];
+
+export type Options = {
+  method: METHODS;
+  data?: any;
+  timeout?: number;
+  headers?: string;
```

Comment:
```suggestion
  headers?: Record<string, string>;
```

Заголовки - это ключ-значение

Reply:
исправил